web-previewer/src/components/usePreviewState.js

```diff
@@ -1,73 +1,87 @@
+      "location",
+      "mind",
+      "os",
+      "texteditor",
+      "vibe",
+      "clothing",
+    ];
 
-        return _layers
-    })
+    let _layers = [];
 
-    function updateDeveloper() {
-        // Loop through all developers until a full trait match is found
-        const _developer = dataDevelopers.find(_developer => {
-            return dataTraits.every(trait => {
-                return _getTraitSlugFromName(trait, _developer[trait.id] ?? null) === traits[trait.slug]
-            })
-        })
+    orderedTraits.forEach((trait) => {
+      if (traits[trait] != null) {
+        _layers.push(`/traits/${trait}/${traits[trait]}.png`);
+      }
+    });
 
-        // Apply developer id. Otherwise reset.
-        developer.value = _developer
-            ? _developer.id
-            : null
+    if (computer.value) {
+      _layers.push(`/traits/computer.png`);
     }
 
-    function updateTraits() {
-        // Find developer matching id.
-        const _developer = dataDevelopers.find(dev => parseInt(dev.id) === parseInt(developer.value))
+    return _layers;
+  });
 
-        // If found developer from current id, apply trait values. Otherwise reset.
-        dataTraits.forEach(trait => {
-            traits[trait.slug] = _developer
-                ? _getTraitSlugFromName(trait, _developer[trait.id] ?? null)
-                : null
-        })
-    }
+  function updateDeveloper() {
+    // Loop through all developers until a full trait match is found
+    const _developer = dataDevelopers.find((_developer) => {
+      return dataTraits.every((trait) => {
+        return (
+          _getTraitSlugFromName(trait, _developer[trait.id] ?? null) ===
+          traits[trait.slug]
+        );
+      });
+    });
 
-    function _getTraitSlugFromName(trait, name) {
-        return trait.values.find(value => value.name === name)?.slug
-    }
+    // Apply developer id. Otherwise reset.
+    developerId.value = _developer ? _developer.id : null;
+  }
 
-    // Initialize
-    dataTraits.forEach(trait => traits[trait.slug] = null)
+  function updateTraits() {
+    // Find developer matching id.
+    const _developer = dataDevelopers.find(
+      (dev) => parseInt(dev.id) === parseInt(developerId.value)
+    );
 
-    updateTraits()
+    // If found developer from current id, apply trait values. Otherwise reset.
+    dataTraits.forEach((trait) => {
+      traits[trait.slug] = _developer
+        ? _getTraitSlugFromName(trait, _developer[trait.id] ?? null)
+        : null;
+    });
+  }
 
-    return {
-        computer,
-        dataTraits,
-        developer,
-        layers,
-        traits,
-        updateDeveloper,
-        updateTraits,
-    }
-}
\ No newline at end of file
+  function _getTraitSlugFromName(trait, name) {
+    return trait.values.find((value) => value.name === name)?.slug;
+  }
+
+  return {
+    computer,
+    dataTraits,
+    developerId,
+    layers,
+    traits,
+    updateDeveloper,
+    updateTraits,
+  };
+}
```

Comment:
Are there actually made any changes in this file, or can we revert it to the original?

Reply:
No changes here except the `developerId` that will be reverted back to `developer`